het_control/conf/sampling_iddpg_config.yaml

```diff
@@ -13,7 +13,7 @@ use_action_loss: True
 action_loss_lr: 0.00003
 
 experiment:
-  max_n_frames: 5_000_000
+  max_n_frames: 1_000_000
```

Comment:
I would not change the default config for reproducibility

Reply:
sorry, about that I didnt want to run for longer, so pushed this by mistake.